Composer/packages/client/src/pages/botProject/adapters/ExternalAdapterSettings.tsx

```diff
@@ -133,31 +101,11 @@ const ExternalAdapterSettings = (props: Props) => {
                 }}
               />
             </div>
-            <TooltipHost content={formatMessage('Actions')} directionalHint={DirectionalHint.rightCenter}>
-              <IconButton
-                ariaLabel={formatMessage('Actions')}
-                className="dialog-more-btn"
-                data-testid="dialogMoreButton"
-                menuIconProps={{ iconName: 'MoreVertical' }}
-                menuProps={{
-                  items: [
-                    {
-                      key: 'edit',
-                      text: formatMessage('Edit'),
-                      iconProps: { iconName: 'Edit' },
-                      onClick: () => openModal(key, false, packageName),
-                    },
-                  ],
-                }}
-                role="cell"
-                styles={{ root: { paddingTop: '10px', paddingBottom: '10px' } }}
-                onKeyDown={(e) => {
-                  if (e.key === 'Enter') {
-                    e.stopPropagation();
-                  }
-                }}
-              />
-            </TooltipHost>
+            <div css={tableRowItem(columnSizes[2])} role="cell">
+              <Link key={key} onClick={() => openModal(key, packageName)}>
```

Comment:
can we rename event handler as on{actionName}{eventName}. essentially, maybe onOpenModalClick?